### ISSUE TITLE
Allow plural route naming convention

### ISSUE THREAD

**Issue by pktharindu** [laravel-shift/blueprint#480]
### Synopsis:

First of all, thank you so much for this amazing package! It saves me a ton of time. 

I'm not sure whether this is intentional, but according to Laravel and REST naming conventions, the route URIs should be in plural form.

**Refferences:** 

- https://github.com/alexeymezenin/laravel-best-practices#follow-laravel-naming-conventions
- https://restfulapi.net/resource-naming/
- https://laravel.com/docs/8.x/controllers#resource-controllers

### Expected Behavior:

Currently, the generated routes are in the singular form. For example:

```yml
controllers:
  Post:
    resource: api
```
Generates:
```php
Route::apiResource('post', App\Http\Controllers\PostController::class);
```
Instead, it should be:
```php
Route::apiResource('posts', App\Http\Controllers\PostController::class);
```

**Comment by jasonmccreary** [comment]
I am open to adding a configuration option for generating plural routes, view, etc. However, per the previous PR, this is not a small task. I will leave this open for the community to implement if wanted.

**Comment by jasonmccreary** [comment]
Merged in #574
